examples/demo.js

```diff
@@ -55,7 +61,7 @@ window.onload = function () {
   // UI for the overall demo:
   var ui = defaultHtmlSequencerUi(sequencer);
 
-  // find any `src` parameters in URL hash and attempt to source image from them and run the sequencer
+  // Load image data from URL `src` parameter.
```

Comment:
You missed some information here from ealier comments

Reply:
That info is not reqd.